recipes/sail/all/conanfile.py

```diff
@@ -18,25 +18,21 @@ class SAILConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "thread_safe": [True, False],
-        "with_avif": [True, False],
```

Comment:
It's not clear what's high/medium/low priority, any option should be easy to understand, that's why we follow the upstream options, or try to add something known. I would ask to revert it and keep the library name instead. In case some of them are not compatible to be built together, then you can use `validate()` to check and raise an `ConanInvalidConfiguration`.

Reply:
Hi. Low/high/highest etc. are just sets of codes as implemented in upstream. It's much easier to maintain codecs in sets as SAIL gets more and more of them, and I don't want to end up having 25 different `with_[codec]` options. SAIL still supports enabling or disabling individual codecs along with sets. Please share what you think about it. If you think individual `with_[codec]` options must be kept along with sets, I will update the recipe.

Reply:
I'll talk with Conan team about it. Indeed is much easier having a set of codecs than each codec as an option. This change is a breaking change per say, because changes a bunch of options.

Reply:
Added `_codecs` suffix to the priorities to make priorities a little bit more clear.